website/content/docs/usage/notebook-sample/index.md

```diff
@@ -1,99 +1,170 @@
-  labels:
-    app: my-notebook-app
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: my-notebook-app
-  template:
-    metadata:
-      labels:
-        app: my-notebook-app
-      annotations:
-        sidecar.istio.io/inject: "true"
-    spec:
-      containers:
-        - name: notebook
-          image: jupyter/datascience-notebook
-          ports:
-            - containerPort: 8888
-              protocol: TCP
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: notebook-service
-  labels:
-    app: my-notebook-app
-spec:
-  ports:
-    - protocol: TCP
-      name: http
-      port: 80
-      targetPort: 8888
-  selector:
-    app: my-notebook-app
-  type: ClusterIP
-```
+In this sample guide you will:
+1. Prepare a dataset to be accessed by the notebook
+1. Register the dataset in ODPi Egeria catalog
+1. Provide {{< name >}} with credentials to the dataset
+1. Deploy a Kubeflow notebook
+1. Create a {{< name >}} runtime environment for the notebook
+1. Read the dataset and observe policies applied seamlessly
 
-# Register with {{< name >}}
-
-Your notebook don't have data access credentials and would not be able to operate without registering the Application to the {{< name >}} control plane. 
-
-Register your application by creating a `M4DApplication` resource that provides metadata about your application:
-
-```yaml
-apiVersion: app.m4d.ibm.com/v1alpha1
-kind: M4DApplication
-metadata:
-  name: my-notebook-app
-  labels:
-    app: my-notebook-app
-spec:
-  selector:
-    matchLabels:
-      app: my-notebook-app
-  appInfo:
-    purpose: fraud-detection
-    processingGeography: US
-    role: Security
-  data:
-    - catalogID: 87ffdca3-8b5d-4f77-99f9-0cb1fba1f73f
-      dataSetID: 01c6f0f0-9ffe-4ccc-ac07-409523755e72 
-      ifDetails:
-        protocol: S3
-        dataformat: parquet
-```
+## Getting started
+
+1.  Obtain a local copy of {{< name >}} repository
+    ```bash
+    git clone https://github.com/IBM/the-mesh-for-data.git
+    ```
+1.  Change to the root directory of the repository
+    ```bash
+    cd the-mesh-for-data
+    ```
+
+## Prepare the dataset for the sample notebook
+
+1. Upload [data.csv](https://{{< github_base >}}/{{< github_repo >}}/blob/master/samples/kubeflow/data.csv) to an object storage of your choice
+    `data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data) created by NTNU, and it is shared under the ***CC BY-SA 4.0*** license.
+1. Update ```samples/kubeflow/example_transactions.csv.json``` with the location of the dataset, The location is encoded in the `fullPath` field as follows:
+    - Bucket: Change the `bucket` value from `m4d-bucket-example` to the bucket name where the dataset reside.
+    - endpoint: Change the S3 endpoint name from `s3.eu-de.cloud-object-storage.appdomain.cloud` to the object storage endpoint
+    - object_key: Change object_key from `data.csv` to the object name that holds the dataset.
```

Comment:
But we uploaded data.csv. Maybe explicitly say that this is only needed if you used another file.

Reply:
should be there in case the object name is not data.csv, reflected that in the text